packages/tpetra/core/src/Tpetra_BlockMultiVector_decl.hpp

```diff
@@ -138,6 +138,11 @@ namespace Tpetra {
 /// It doesn't make sense for BlockMultiVector to implement
 /// MultiVector, because the desired fill interfaces of the two
 /// classes are different.
+///
+/// BlockMultiVector does not have comm layer e.g., packAndPrepare, 
```

Comment:
Hm, this wasn't what I meant.  What I was thinking is that we could implement BlockMultiVector's DistObject methods by dispatching to MultiVector methods on the underlying MultiVector.  That would also simplify BlockCrsMatrix -- BlockCrsMatrix could just trust BlockMultiVector to do the right thing, and BlockMultiVector would call MultiVector's methods.

Reply:
I am not sure if that approach is feasible. I think that what you want, for example ``copyAndPermute``, is 
```
BlockMultiVector(src, numSameIDs, permuteToLIDs, permuteFromLIDS) {
  this->getMultiVectorView().copyAndPermute
    (src.getMultiVectorView(),
     numSamdIDs*blockSize_,
     // the following is problematic
     permuteToLIDs.withMagicToPopulateIndicies(),
     permuteFromLIDS.withMagicToPopulateIndicies());
 }
```
If this is what you intended, it is a bit problematic as it requires a new view object to populate flattened indicies. I can write the routine with some cost and we can recommend not to use block multivector importer. However, I do not think that this is good approach. That is why I completely remove the comm layer. Instead of deleting the methods, I can put the less-performant version with a warning message to make the block multivector comm operational.

Reply:
Ah, good point, I forgot about the indexing thing.  I'm OK with this.

Reply:
I would like to do so but I see some missing symbol errors in linking. I may need to provide some fake imple with throwing an error if it is used. I will also modify the test to use the point multi vector dist object interface.